tests/test_library_vmaf_eval.py

```
@@ -0,0 +1,358 @@
+        pytest.skip("PowerShell not available")
+    config = tmp_path / "vmaf-eval-config.json"
+    config.write_text(json.dumps({
+        "pairs": [
+            {
+                "video": "clip1",
+                "source": str(FIXTURE_DIR / "clip1.mp4"),
+                "variant": str(FIXTURE_DIR / "clip1_low.mp4"),
+            },
+        ]
+    }))
+    result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)